packages/config/src/load.js

```diff
@@ -117,6 +118,23 @@ export async function loadNuxtConfig ({
     }
   }
 
+  if (options.publicRuntimeConfig) {
```

Comment:
- This logic could be better in `options.js` after applying defaults
- We may set value of `options.router.base` to `publicRuntimeConfig._nuxt.routerBase` so **not introducing a new option** but changing router.base in production applies to runtime
- As you suggested it might be better to use `nuxt` / `_nuxt` namespace or something unique inside `$config`
- We can simplify names to something like `routerBase` and `publicPath` in nuxt's runtimeConfig namespace since they are internal for directly reading from `$config`

Reply:
Great idea. That lets us simplify this considerably. I've left the logic in `load.js` for the moment (thinking to ensure we have the last say on content of `_nuxt`) but happy to move into `options.js` if you feel that's more logic.